### ISSUE TITLE
Bug: ...Windscribe missing location US East

### ISSUE THREAD

**Issue by udithap** [qdm12/private-internet-access-docker#198]
**TLDR**: *US Central and US West is mentioned as possible values for region, US East is inexplicably missing*

1. Is this urgent?

    - [ ] Yes
    - [x] No

2. What VPN service provider are you using?

    - [ ] PIA
    - [ ] Mullvad
    - [x] Windscribe
    - [ ] Surfshark
    - [ ] Cyberghost

3. What's the version of the program?

    **See the line at the top of your logs**

    `Running version latest built on 2020-03-13T01:30:06Z (commit d0f678c)`

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

Logs:

```log

```

Configuration file:

```yml

```

Host OS:

**Comment by qdm12** [comment]
Yes that's a recent change, for some reason my program would not want to resolve us-east.windscribe.com to ip addresses. Anyway, I've done manually so it should be back. Thanks for letting me know.
